src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultControllerModelBuilder.cs

```diff
@@ -56,6 +57,16 @@ public ControllerModel BuildControllerModel([NotNull] TypeInfo typeInfo)
                 }
             }
 
+            foreach (var propertyInfo in controllerType.GetProperties(BindingFlags.Public | BindingFlags.Instance))
```

Comment:
Suggest factoring this into a `protected virtual IsBindableProperty(...)` kind of method. IIRC most of the model builder code is already extensible in this kind of way.